recipes/warp-lang/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "warp-lang" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/warp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fc80bc6c38ac2e0a1a59f9dc912a29692bcf3db2848625890fbcdac45b2c06c9
+  patches:
+    - 01-find_g++_env.patch
+    - 02-add_macro_def_build.patch
+    - 03-install_deps_llvm_conda.patch
+    
+build:
+  script: 
+    - {{ PYTHON }} build_lib.py
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-cache-dir  
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Does this need NVCC to build?

In the docs it mentions needing [CUDA Toolkit]( https://github.com/NVIDIA/warp#building ). It also appears to [look for the CTK]( https://github.com/NVIDIA/warp/blob/79a56a9c2754bbec1418e490efafd43091056856/build_lib.py#L74-L79 )

```suggestion
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}
```

---

If we want GPU support yes, for the time being I can stick to the CPU version.